### ISSUE TITLE
Error on video playback in trailers

### ISSUE THREAD

**Issue by zany130** [ValveSoftware/steam-for-linux#8313]
#### Your system information
https://gist.github.com/zany130/e0d01f4a7c28ce4d138efae1a37b55da
* Steam client version (build number or date): Dec 16
* Distribution (e.g. Ubuntu): Garuda Linux (arch based)
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:

Issue: Videos of game trailers sometimes fail to play with a black screen that says error occured during playback

Expected: Videos play normally
#### Steps for reproducing this issue:

1. set screen saver for big picture to TrailerTV or open a game's store page
2. try to watch video in store page or let screensaver play
3. See error

**Comment by kisak-valve** [comment]
Hello @zany130, can you give some examples of Steam store videos that fail to play for you?

**Comment by zany130** [comment]
This issues has been happening to me for a while and I thought there was already an open issue but couldn't find it. Don't remember which store page videos were having issues for me. I see the issue more in the TrailerTV screensaver I'll see if I can get a name of a game that gives that error and then it might be interesting to see if i can play the same games store page trailers

**Comment by kisak-valve** [comment]
I can reproduce this issue with The Talos Principle VR and Half-Life - The Freeman Chronicles. Store videos work in Large mode, but not Big Picture mode.

We'll need a Steam dev to take a closer look at what the difference is between those and the majority of game trailers.

**Comment by zany130** [comment]
Yep tried those games to i get the error occured during playback on there store page
